File: python/calculator/calculator/calculator.py
from playwright.sync_api import sync_playwright

CALCULATOR_URL = "https://treasurydirect.gov/BC/SBCPrice"

with sync_playwright() as p:
    browser = p.firefox.launch()
    page = browser.new_page()
    page.goto(CALCULATOR_URL)
    print(page.title())
    browser.close()

File: typescript/src/calculator.js
import { firefox } from "playwright";
import assert from "node:assert";

const CALCULATOR_URL = "https://treasurydirect.gov/BC/SBCPrice";

(async () => {
  const browser = await firefox.launch();
  const context = await browser.newContext();
  const page = await context.newPage();
  await page.goto(CALCULATOR_URL);
  await context.close();
  await browser.close();
})();
